audio.go

```diff
@@ -89,3 +105,87 @@ func (a *App) unloadMusic() {
 		rl.UnloadMusicStream(*a.music)
 	}
 }
+
+// Sound source type
+type Sound = rl.Sound
+
+// LoadSoundFromFile - Load sound stream from a file path
+func LoadSoundFromFile(path string) *Sound {
+	sound := rl.LoadSound(path)
+	if !rl.IsSoundReady(sound) {
+		log.Fatalf("Failed to load sound from file %s", path)
+	}
+	return &sound
+}
+
+// SetSoundVolume - Set volume for sound (1.0 is max level)
+func (a *App) SetSoundVolume(volume float32) {
+	if a.isSoundReady() {
+		rl.SetSoundVolume(*a.sound, volume)
+	}
+}
+
+func (a *App) isSoundReady() bool {
+	return a.sound != nil && rl.IsSoundReady(*a.sound)
+}
+
+// SoundPlay is a command that will play the sound with the given resource locator.
+type SoundPlay struct {
+	SoundResource ResourceLocator
+}
+
+func (cmd SoundPlay) Execute(app *App, done Promise) {
+	app.sound = app.res.LoadSound(cmd.SoundResource)
+	if app.isSoundReady() {
+		rl.PlaySound(*app.sound)
+	}
+	done.Complete()
+}
+
+// SoundStop is a command that will stop the sound with the given resource locator.
+type SoundStop struct {
+	SoundResource ResourceLocator
+}
+
+func (cmd SoundStop) Execute(app *App, done Promise) {
+	app.stopSound()
+	done.Complete()
+}
+
+// SoundPause is a command that will pause the sound with the given resource locator.
+type SoundPause struct {
+	SoundResource ResourceLocator
+}
+
+func (cmd SoundPause) Execute(app *App, done Promise) {
+	app.pauseSound()
+	done.Complete()
+}
+
```

Comment:
If the difference between music and sound is much sorter (used for effects), why do we need to pause or resume it?

Reply:
You are right, lets simplify the API removing these functions

Reply:
Solved here: https://github.com/apoloval/pctk/pull/7